### PR TITLE
how to run

### PR DESCRIPTION
**Pull request by laurentdebricon**
lost 5 min how to run :D

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Require the [go programming language](https://golang.org/) (version >= 1.13)
 ```
 git clone https://github.com/edouardparis/lntop.git
 cd lntop && go install ./...
+go build -o lntop cmd/lntop/main.go
```
Comment (edouardparis) [inline README.md:22]: Can you separate it from the go install command in a new code snippet like this:

--

If you do not want to have the binary inside your $GOPATH, you can create it and run it inside the repository:
```
go build -o lntop cmd/lntop/main.go
./lntop
```

**Comment by edouardparis** [comment]
Sorry about that, `go install ./...` creates the binary inside your GOPATH (generally `~/go/bin`) and you have to add `export PATH="$PATH:$HOME/go/bin"` to you .bashrc.

**Comment by laurentdebricon** [comment]
i think i close this pull request, and when you have time, you improve the readme for noobs like me. thxx !

**Comment by edouardparis** [comment]
Thank you you for the issue, I will help other users !

**Pull request closed without merging**